Coalition.rte/Devices/Weapons/GatlingGun/MuzzleSmoke.lua

```diff
@@ -0,0 +1,22 @@
+function Create(self)
```

Comment:
This script seems exactly like the one above for Gattling Drone? Why not just use one general Gattling Script? Also, why are the effects being done through lua anyway, instead of emitters or whatever?